jest/hasteImpl.js

```diff
@@ -57,8 +57,8 @@ const WHITELISTED_PREFIXES /*: Array<string> */ = [
 const NAME_REDUCERS /*: Array<[RegExp, string]> */ = [
   // extract basename
   [/^(?:.*[\\\/])?([a-zA-Z0-9$_.-]+)$/, '$1'],
-  // strip .js/.js.flow suffix
-  [/^(.*)\.js(\.flow)?$/, '$1'],
+  // strip .js/.ts/.tsx/.js.flow suffix
```

Comment:
I don't think the changes to this file are necessary, are they? This only affects the react-native repo itself.

Reply:
@cpojer , you are right. I fixed it in https://github.com/facebook/react-native/pull/22217/commits/62551c2162e60b1a63c037089ae59b833602ebf0